docs/OnlineTutorial/Lemmas.md

```diff
@@ -522,35 +516,29 @@ if statement:
 
 ``` {.editonly}
 lemma DistributiveLemma(a: seq<bool>, b: seq<bool>)
-   ensures count(a + b) == count(a) + count(b)
-{
-   if a == []
-   {
-      assert a + b == b;
-   }
-   else
-   {
-      DistributiveLemma(a[1..], b);
-      assert a + b == [a[0]] + (a[1..] + b);
-   }
+  ensures count(a + b) == count(a) + count(b)
+{
+  if a == [] {
+    assert a + b == b;
+  } else {
+    DistributiveLemma(a[1..], b);
+    assert a + b == [a[0]] + (a[1..] + b);
+  }
 }
 function count(a: seq<bool>): nat
 {
-   if |a| == 0 then 0 else
-   (if a[0] then 1 else 0) + count(a[1..])
+  if |a| == 0 then 0 else
+    (if a[0] then 1 else 0) + count(a[1..])
```

Comment:
```suggestion
  if |a| == 0 then 0 else
  (if a[0] then 1 else 0) + count(a[1..])
```

Reply:
Are you sure about this? The second line is not a new if statement, but the value for the else. I would indent it.